lib/src/ui/common/widget/retry_action_container.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:allthenews/generated/l10n.dart';
+import 'package:allthenews/src/ui/common/widget/primary_text_button.dart';
+import 'package:flutter/material.dart';
+
+class _Constants {
+  static const retryButtonWidth = 120.0;
+  static const retryButtonHeight = 40.0;
+}
+
+class RetryActionContainer extends StatelessWidget {
+  final VoidCallback onRetryPressed;
+
+  const RetryActionContainer({@required this.onRetryPressed}) : assert(onRetryPressed != null);
+
+  @override
+  Widget build(BuildContext context) => Center(
+        child: SizedBox(
+          height: _Constants.retryButtonHeight,
+          width: _Constants.retryButtonWidth,
+          child: PrimaryTextButton(
+            onPressed: () => onRetryPressed,
```

Comment:
nie wiem z czego to wynika, ale jak się wchodzi do apsy bez neta, a potem go włącza i klika `Retry` to nic się nie dzieje